<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1206</width>
    <height>709</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>11</pointsize>
   </font>
  </property>
  <property name="cursor">
   <cursorShape>PointingHandCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*{
background:#A7CECB;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1211</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sitka</family>
      <pointsize>24</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#CACC90;</string>
    </property>
    <property name="text">
     <string>GajiKu</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Nama Karyawan :</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="EditNama">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>130</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="EditGajiPokok">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>250</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Jabatan :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>250</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Gaji Pokok</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>310</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Tahun Masuk Kerja :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>430</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Tahun Habis Kontrak :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>370</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Status Kerja :</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Btntambah">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>500</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>Tambah</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>C:/Users/Yusa8/Downloads/ICONS/add.png</normaloff>C:/Users/Yusa8/Downloads/ICONS/add.png</iconset>
    </property>
   </widget>
   <widget class="QTableWidget" name="tabelgaji">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>180</y>
      <width>741</width>
      <height>451</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="cursor" stdset="0">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background:#E7F6F2</string>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::SingleSelection</enum>
    </property>
    <property name="selectionBehavior">
     <enum>QAbstractItemView::SelectRows</enum>
    </property>
    <column>
     <property name="text">
      <string>namakaryawan</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>jabatan</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>gajipokok</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>thnmasuk</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>statuskerja</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>thhbskontrak</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="Btnperbarui">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>500</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>Perbarui</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>C:/Users/Yusa8/Downloads/ICONS/update.png</normaloff>C:/Users/Yusa8/Downloads/ICONS/update.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="Btnhapus">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>560</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>Hapus</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>C:/Users/Yusa8/Downloads/ICONS/delete.png</normaloff>C:/Users/Yusa8/Downloads/ICONS/delete.png</iconset>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>120</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Cari Nama Karyawan :</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="carinamakaryawan">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>120</y>
      <width>491</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Btncari">
    <property name="geometry">
     <rect>
      <x>1090</x>
      <y>120</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>Cari</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>C:/Users/Yusa8/Downloads/ICONS/search.png</normaloff>C:/Users/Yusa8/Downloads/ICONS/search.png</iconset>
    </property>
   </widget>
   <widget class="QLineEdit" name="EditTahunMasuk">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>310</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="EditJabatan">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>190</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="EditStatusKerja">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>370</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="TahunhabisKontrak">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>430</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
